src/style.css

```diff
@@ -1,3 +1,73 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  font-family: Roboto, serif;
```

Comment:
Remember to use fallback fonts - alternative font-family in case the main one doesn't work. For example: `font-family: Roboto, Arial, sans-serif;`. (Checklist: Remember to use fallback fonts)